superset-frontend/src/components/CertifiedBadge/index.tsx

```diff
@@ -41,7 +41,9 @@ function CertifiedBadge({
         <>
           {certifiedBy && (
             <div>
-              <strong>{t('Certified by %s', certifiedBy)}</strong>
+              <strong>
+                {t('Certified by %s', ensureIsArray(certifiedBy).join(', '))}
```

Comment:
As Tom mentioned, will this work with a single value from an existing record?

Reply:
@cccs-joel I looked up the code and that's exactly what `ensureIsArray` does (it also handles `null` values and returns an empty array). I just think we need test coverage for this. 😄 
And speaking of null checks, I don't know if the value of this field is ever compared against null. But if it is, the condition would have to be updated to also include an empty list, imo.

Reply:
Great!